### ISSUE TITLE
cannot load such file error.  /lib/codecov/version

### ISSUE THREAD

**Issue by tknzk** [codecov/codecov-ruby#119]
```
Failure/Error: require "codecov"

LoadError:
  cannot load such file -- /app/vendor/bundle/ruby/2.6.0/gems/codecov-0.2.14/lib/codecov/version
# ./vendor/bundle/ruby/2.6.0/gems/codecov-0.2.14/lib/codecov.rb:9:in `require_relative'
# ./vendor/bundle/ruby/2.6.0/gems/codecov-0.2.14/lib/codecov.rb:9:in `<top (required)>'
```

**Comment by jasonbosco** [comment]
Was just about to open an issue myself. I only see this issue in `v0.2.14`. I downgraded it to `v0.2.13` and it works fine.

**Comment by meganemura** [comment]
#118 fixes this issue.

**Comment by thomasrockhu** [comment]
This should be fixed via `0.2.15`

**Comment by AlexWayfer** [comment]
I'm sorry, it happened after my changes, I didn't notice this line and it didn't affect CI.

I've created #123 with a try to prevent such errors in future in one of commits.
